ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -235,3 +272,64 @@ Set Expected Items Based On RHODS Type    # robocop: disable
     Set Suite Variable    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}
     Set Suite Variable    ${EXPECTED_ITEMS_FOR_COMBINATIONS}    ${EXPECTED_ITEMS_FOR_COMBINATIONS}
 
+Set Quick Starts Elements List Based on RHODS Type
+    [Documentation]    Get QuickStarts list based on RHODS
+    ...     is self-managed or managed
+    ${is_self_managed}=    Is RHODS Self-Managed
+    IF    ${is_self_managed} == ${TRUE}
+            @{quickStartElements}=      Create List
+            ...    connect-rhosak-notebook    create-jupyter-notebook    create-jupyter-notebook-anaconda
+            ...    deploy-python-model    create-aikit-notebook    openvino-inference-notebook
+            ...    pachyderm-beginner-tutorial-notebook    build-deploy-watson-model
+    ELSE
+            @{quickStartElements}=      Create List
+            ...    connect-rhosak-notebook    create-jupyter-notebook    create-jupyter-notebook-anaconda
+            ...    deploy-python-model    create-aikit-notebook    deploy-model-rhoam    gpu-enabled-notebook-quickstart
+            ...    pachyderm-beginner-tutorial-notebook    openvino-inference-notebook    using-starburst-galaxy
+            ...    gpu-quickstart    build-deploy-watson-model
+    END
```

Comment:
could you add a check on the actual number of quick starts retrieved in the dashboard, and the lenght of the manually set lists? If fails, it tells us if a new quickstart has been added or removed

Reply:
Added the step to check the quickstart list.